src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -2,46 +2,44 @@
 
 [Fluent migrations](https://firefox-source-docs.mozilla.org/l10n/migrations/index.html) are used to migrate existing translations from legacy formats (DTD, properties) to Fluent, or existing Fluent messages to new messages or different FTL files.
 
-At the core of each migration there is a Python file, called **migration recipe**, that instructs the system on how to port existing translations to Fluent. Typically, a patch migrating content to Fluent will:
+At the core of each migration there is a Python file, called a **migration recipe**, that instructs the system on how to port existing translations to Fluent. Typically, a patch migrating content to Fluent will:
 * Add new Fluent strings or files.
 * Remove obsolete strings.
 * Include a migration recipe.
 
-Once such patch lands in `mozilla-central`, it will be converted as part of the cross-channel generation, and pushed to a quarantine repository (`gecko-strings-quarantine`). At this point, this new content is not visible yet in [Pontoon](https://pontoon.mozilla.org/).
+Once one such patch lands in [gecko-dev](https://github.com/mozilla/gecko-dev), string changes will be pushed to a quarantine repository (the `update` branch of [firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source)). At this point, this new content is not visible yet in [Pontoon](https://pontoon.mozilla.org/).
```

Comment:
```suggestion
Once one such patch lands in [gecko-dev](https://github.com/mozilla/gecko-dev), string changes will be pushed to the `update` branch of [firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source). At this point, this new content is not visible yet in [Pontoon](https://pontoon.mozilla.org/).
```

It's not a quarantine repository anymore, in case a "quarantine area".